nixos/tests/open-webui.nix

```diff
@@ -18,6 +18,8 @@ in
           environment = {
             # Requires network connection
             RAG_EMBEDDING_MODEL = "";
+
+            WEBUI_URL = "http://localhost:${mainPort}";
```

Comment:
adding this in the test is not ideal, because it overrides the default value

but i didn't found another way while keeping `RAG_EMBEDDING_MODEL`